fieldservice_sale_recurring/readme/INSTALL.md

```diff
@@ -0,0 +1,5 @@
+To install Field Service and have the mapping features, you need to
+install GeoEngine.
+
+Please refer to the installation instructions available at:
+<https://github.com/OCA/geospatial/tree/13.0/base_geoengine>
```

Comment:
Please remove the file, it has nothing to do with the fieldservice_sale_recurring addon.